HOWTOADDSERVICE.md

```diff
@@ -8,9 +8,28 @@ The following values are required to add the configuration for a new service.
 | -------------           | ----------------------- | 
 | `enrolment`             | The enrolment key for the new service.  Details for creating a new service enrolment key can be found [here](https://github.com/hmrc/service-enrolment-config). | 
 | `shortName`             | This is the abbreviation of the service. | 
-| `callBack`              | This is the url ECC will re-direct the user to once they have completed an enrolment request. | 
+| `callBack`              | ECC will re-direct the user to this url to once they have the requested enrolment. | 
```

Comment:
Probably it's worth adding `(see below)` in that case too

Reply:
Good point.  And they can be a link as well.